dashboard.go

```diff
@@ -26,10 +26,11 @@ type Dashboard struct {
 	Model map[string]interface{} `json:"dashboard"`
 }
 
-func (c *Client) SaveDashboard(model map[string]interface{}, overwrite bool) (*DashboardSaveResponse, error) {
+func (c *Client) SaveDashboard(model map[string]interface{}, overwrite bool, folderId int) (*DashboardSaveResponse, error) {
```

Comment:
What is the behaviour when this is not specified?

Reply:
folderId is returned as 0, and that creates the dashboard on the rood folder of grafana